cmake/modules/Finddpdk.cmake

```diff
@@ -26,7 +26,7 @@ foreach(c ${components})
       INTERFACE_INCLUDE_DIRECTORIES "${DPDK_INCLUDE_DIR}"
       IMPORTED_LOCATION "${DPDK_rte_${c}_LIBRARY}")
     target_compile_options(dpdk::${c} INTERFACE
-      "-march=native")
+      "-march=core2")
```

Comment:
@smithfarm after a second thought , i think, probably we should not have this flag at all.

this compile flag was originally introduced in eae7d5a6450aba35ff88e6667c5f2bcca51fe6f8 , and then i constrained the scope of this flag to `common_async_dpdk` target in dbe04b4a85de5d8d68da05f8d1a18af45cacbc78. and in be300380cec704b7337e7cb5137d29039067dd0f, it was then transformed to an INTERFACE compile option of dpdk libraries, in hope to populate this option to the target(s) that links against dpdk libraries. it does work.

but do we really need to make this decision on behalf of our user? i think they should have the liberty to choose whatever `-march=<arch>` they feel is appropriate by specifying the `CMAKE_C_FLAGS` or `CFLAGS` at configure phase.

so, in short, i'd recommend just remove this line.